processor/metricstransformprocessor/README.md

```diff
@@ -55,6 +58,9 @@ transforms:
       aggregated_values: [values...]
       new_value: <new_value> 
       aggregation_type: {sum, mean, max}
+
+    # convert_resource_attributes_to_labels action converts all the resourde attributes to metric labels by default. Right now, it doesn't support selecting specific resource attributes as metric labels.
```

Comment:
Ditto file an issue or update existing one to track the addition.

Reply:
Created an issue: #1343